src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +76,22 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email", methods={"POST"})
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer, SessionInterface $session)
+    {
+        $emailExists = $request->request->has('email');
+        if ($emailExists === true) {
```

Comment:
You can do directly `if ($request->request->has('email'))`, the `$emailExists` is not used anywhere else.